WireDomain/Tests/WireDomainTests/Event Processing/ConversationEventProcessor/ConversationMemberLeaveEventTests.swift

```diff
@@ -0,0 +1,86 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+@testable import WireAPI
+@testable import WireDomain
+import WireDomainSupport
+import XCTest
+
+final class ConversationMemberLeaveEventProcessorTests: XCTestCase {
+
+    private var sut: ConversationMemberLeaveEventProcessor!
+    private var repository: MockConversationRepositoryProtocol!
+
+    override func setUp() async throws {
+        try await super.setUp()
+        repository = MockConversationRepositoryProtocol()
+        sut = ConversationMemberLeaveEventProcessor(
+            repository: repository
+        )
+    }
+
+    override func tearDown() async throws {
+        try await super.tearDown()
+        repository = nil
+        sut = nil
+    }
+
+    // MARK: - Tests
+
+    func testProcessEvent_It_Invokes_Remove_Members_Repo_Method() async throws {
+        // Mock
+
+        repository.removeMembersFromInitiatedByAtReason_MockMethod = { _, _, _, _, _ in }
+
+        // When
+
+        try await sut.processEvent(Scaffolding.event)
+
+        // Then
+
```

Comment:
don't we have an extra line space here, curious why swiftlint did not catch it here. Is it configured properly in WireDomain?

Reply:
no extra line space